scipy/linalg/decomp.py

```diff
@@ -771,66 +859,99 @@ def eigvals(a, b=None, overwrite_a=False, check_finite=True,
 
 def eigvalsh(a, b=None, lower=True, overwrite_a=False,
              overwrite_b=False, turbo=True, eigvals=None, type=1,
-             check_finite=True):
+             check_finite=True, subset_by_index=None, subset_by_value=None,
+             driver=None):
     """
-    Solve an ordinary or generalized eigenvalue problem for a complex
+    Solves a standard or generalized eigenvalue problem for a complex
```

Comment:
I didn't compare this docstring line by line with `eigh`, but I assume they are consistent.

Reply:
I think it should be "Solve" though.

Reply:
Well I didn't change it : ) do you want me to compare carefully?

Reply:
No just realized my own mistake